agent/consul/stream/structs.proto

```diff
@@ -0,0 +1,377 @@
+syntax = "proto3";
+
+package stream;
+
+import "google/protobuf/duration.proto";
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+option (gogoproto.goproto_unkeyed_all) = false;
+option (gogoproto.goproto_unrecognized_all) = false;
+option (gogoproto.goproto_getters_all) = false;
+option (gogoproto.goproto_sizecache_all) = false;
+
+// Used to return information about a node
+message Node {
+  string ID = 1
+      [(gogoproto.casttype) = "github.com/hashicorp/consul/types.NodeID"];
+
+  string Node = 2;
+  string Address = 3;
+  string Datacenter = 4;
+  map<string, string> TaggedAddresses = 5;
+  map<string, string> Meta = 6;
+
+  RaftIndex RaftIndex = 7
+      [(gogoproto.embed) = true, (gogoproto.nullable) = false];
+}
+
+// ServiceDefinition is used to JSON decode the Service definitions. For
```

Comment:
Is this copy pasta from the original struct? Maybe if this file was generated we should add a comment to the top to make it clear?